packages/discovery-react-components/src/components/ResultsPagination/ResultsPagination.tsx

```diff
@@ -126,17 +167,19 @@ const ResultsPagination: FC<ResultsPaginationProps> = ({
       <>
         {!isResultsPaginationComponentHidden && (
           <CarbonPagination
-            className={classNames.join(' ')}
-            initialPage={currentPage}
```

Comment:
Why remove `initialPage`?

Reply:
We weren't using it right-- it's a single-use prop that [sets the initial value of a state variable](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Pagination/experimental/Pagination.js#L39) in Carbon's experimental Pagination component. It's for cases where you want to load pagination on a page other than the first (i.e. the value defaults to 1). This was a key reason why our pagination wasn't working-- I think we assumed that this prop would set the page number as we changed the value, but it simply doesn't.